src/libs/ComposerUtils/types.ts

```diff
@@ -1,6 +1,9 @@
 type ComposerProps = {
```

Comment:
I'm curious about this file, we are going to have two `ComposerProps` now? What's the difference between them?

Reply:
My bad, I forgot to replace it for android, update incoming

Reply:
Updated